app/forms/green_lanes/applicable_exemptions_form.rb

```diff
@@ -38,7 +38,7 @@ def initialize(attributes)
     end
 
     def exempt?
-      answers.values.none? { |value| value == NONE_ANSWER_WAS_GIVEN }
+      !answers.values.flatten.include?(NONE_ANSWER_WAS_GIVEN)
```

Comment:
Changed this because sometimes if multiple answers we were getting `['','','none']` so if it contains none it counts as a none answer